exporter/otlpexporter/testdata/invalid_configs.yaml

```diff
@@ -53,3 +53,19 @@ invalid_retry:
     multiplier: 1.3
     max_interval: 60s
     max_elapsed_time: 10m
+missing_port:
+  endpoint: example.com
```

Comment:
Would it make sense to test an endpoint using a wrong port? Like `example.com:stuff` instead of `example.com:8080`

Reply:
Addressed in latest push. Resolving